models/chirp.js

```diff
@@ -2,9 +2,41 @@ const mongoose = require('mongoose');
 
 //Schema Config
 var chirpSchema = new mongoose.Schema({
-    body: String,
-    //emoji: ?
-    //photo: ?
+    body: {
+        type: String,
+        minlength: 1,
+        maxlength: 280
+    },
+    created: {
+        type: Date,
+        default: Date.now
+    }, 
+    deleted: {
+        type: Boolean, 
+        default: 0
+    },
```

Comment:
Since this is the last uncommented attribute of the chirpSchema, it shouldn't have a comma after it.

Reply:
Noted.